cli/ballerina-cli/src/main/resources/cli-help/ballerina-grpc.help

```diff
@@ -21,11 +22,13 @@ OPTIONS
            definition. If a package is not specified, the output will be written
            to a 'temp' directory in the current location.
 
-       --mode client | service | proxy
+       --mode client | service
            Set the `client` or `service` mode to generate sample code.
```

Comment:
```suggestion
           Set the 'client' or 'service' mode to generate sample code.
```

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/34686/commits/0ba70265ce04e785ee7394104e66b7355e09ef49